src/Random.savi

```diff
@@ -181,3 +181,18 @@
     )-\(
       @u32.format.hex.bare)\(@u16.format.hex.bare
     )"
+
+  :: Return an arbitrary number of random bytes.
+  :fun ref random_bytes(size USize) Bytes
+    bytes = Bytes.new_iso(size)
+
+    while (size > 0) (
+      case size > (
+      | 8 | bytes.push_native_u64(@u64), size -= 8
+      | 4 | bytes.push_native_u32(@u32), size -= 4
+      | 2 | bytes.push_native_u16(@u16), size -= 2
+      |     bytes.push(@u8),             size -= 1
+      )
+    )
```

Comment:
I love the embracing of table-like alignment here - it looks so nice and easy to read!